Admin/AdminHelper.php

```diff
@@ -335,6 +334,8 @@ private function pathExists(PropertyAccessorInterface $propertyAccessor, $entity
             return true;
         } catch (NoSuchPropertyException $e) {
             return false;
+        } catch (UnexpectedTypeException $e) {
```

Comment:
// Symfony <= 2.3 did not have isReadable method for PropertyAccessor
    if (method_exists($propertyAccessor, 'isReadable')) {
        return $propertyAccessor->isReadable($entity, $path);
    }

    try {
        $propertyAccessor->getValue($entity, $path);

        return true;
    } catch (NoSuchPropertyException $e) {
        return false;
    } catch (UnexpectedTypeException $e) {
        return false;
    }

It is already commented in the function. Should I add more information?